polaris-react/playground/Learner.tsx

```diff
@@ -1,31 +1,36 @@
 import React from 'react';
 
-import {Layout, Page, Card, List, InContextLearning, Stack} from '../src';
+import {
+  Layout,
+  Page,
+  Card,
+  List,
+  InContextLearning,
+  InContextLearningContextProvider,
+  Stack,
+} from '../src';
 
 import styles from './Learner.scss';
 
-const LEARNING_STEPS = [
-  {
-    selector: '[data-learning-step-one]',
-    content: <span>Step 1: Testing React</span>,
-  },
-  {
-    selector: '[data-learning-step-two]',
-    content: <span>Step 2: Testing React</span>,
-  },
-  {
-    selector: '[data-learning-step-three]',
-    content: <span>Step 3: Testing React</span>,
-  },
-];
+function StepOne() {
+  return <span>Step 1: Testing React</span>;
```

Comment:
just curious, why are the steps a `React.FunctionComponent` vs a plain ole ReactNode?

Reply:
I'm still somewhat newish to Typescript, so I'm happy for suggestions on what is appropriate here. What we're passing along isn't a node, but the function component itself. 

Do you have any thoughts on this approach? I'm not too attached to how this is being done now vs passing along a node.

Reply:
Makes sense! Probably not worth examining the pros in cons for a hack day. Just curious what was the reasoning behind making them functions versus nodes. e.g. `function StepOne() {
 return <span>Step 1: Testing React</span>;
}` would be `const StepOne = <span>Step 1: Testing React</span>;`
and then in the component itself you would just use `{steps[currentStep].component}` instead of `{steps[currentStep].component()}`

Reply:
oops I see @alololox already commented below

Reply:
Yup. This was a good callout. I adjusted the code to use React nodes instead.